File: game.py
from flask import Flask
import random
import subprocess

app = Flask(__name__)

@app.route('/run-python')
def run_python():
    # لعبة تخمين الرقم بلغة Python
    target = random.randint(1, 10)
    guess = random.randint(1, 10)
    if target == guess:
        return f"تهانينا! الرقم الصحيح هو {target}."
    else:
        return f"خطأ! الرقم الصحيح هو {target}، لكنك اخترت {guess}."

@app.route('/run-cpp')
def run_cpp():
    # تشغيل لعبة C++ باستخدام subprocess
    result = subprocess.run(['./cpp_game'], capture_output=True, text=True)
    return result.stdout

if __name__ == '__main__':
    app.run(debug=True)

File: game.js
let score = 0;

const targetButton = document.getElementById('target-button');
const scoreDisplay = document.getElementById('score');
const message = document.getElementById('message');
const pythonButton = document.getElementById('run-python');
const cppButton = document.getElementById('run-cpp');
const output = document.getElementById('output');

// لعبة النقاط بالـ JavaScript
targetButton.addEventListener('click', () => {
  score++;
  scoreDisplay.textContent = score;

  if (score < 5) {
    message.textContent = "استمر، أنت في البداية!";
  } else if (score < 10) {
    message.textContent = "أنت تتقدم بشكل رائع!";
  } else {
    message.textContent = "مذهل! استمر في اللعب!";
  }
});

// استدعاء لعبة Python
pythonButton.addEventListener('click', async () => {
  const response = await fetch('/run-python');
  const result = await response.text();
  output.textContent = "Python: " + result;
});

// استدعاء لعبة C++
cppButton.addEventListener('click', async () => {
  const response = await fetch('/run-cpp');
  const result = await response.text();
  output.textContent = "C++: " + result;
});
س